tests/tacacs/test_authorization.py

```diff
@@ -219,7 +214,17 @@ def test_authorization_tacacs_only(
         "sonic-db-cli  CONFIG_DB HGET \"FEATURE|macsec\" state"
     ]
 
+    frontend_commands = [
+        "show ip bgp neighbor",
+        "show ipv6 bgp neighbor",
+        "show ip bgp summary",
+        "show ipv6 bgp summary",
+        "show muxcable firmware",
+    ]
+
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+    if duthost.sonichost.is_frontend_node():
```

Comment:
I think it's a good idea to separate the frontend commands.
Can we always choose a frontend dut with enum_rand_one_per_hwsku_frontend_hostname
to make sure the frontend_commands can pass?

Reply:
That will make the scope of the test case more restricted. It is not necessary to do that right?

Reply:
The test currently will run on both the supervisor and linecard each time so we will still have test coverage with the full list of commands during each sonic-mgmt run.